docs/content/docs/connectors/datastream/elasticsearch.md

```diff
@@ -43,11 +43,11 @@ of the Elasticsearch installation:
   </thead>
   <tbody>
     <tr>
-        <td><= 6.3.1</td>
+        <td>6.x</td>
         <td>{{< artifact flink-connector-elasticsearch6 >}}</td>
     </tr>
     <tr>
-        <td><= 7.5.1</td>
+        <td>7.x</td>
```

Comment:
Why do we switch from real versions to only describing the major version?

Reply:
On the last PR, Martijn and me were not sure we could actually support a higher server version than the one we used in the client (v7.5.1 at the time). After the code changes of this PR I manually ran the integration tests to confirm the connector works across all minor versions for the respective major version